Overview.md

```diff
@@ -0,0 +1,65 @@
+
+# Confidential Containers
+We are interested in integrating existing [Trusted Execution Environments](https://en.wikipedia.org/wiki/Trusted_execution_environment) (TEE) 
+infrastructure support and technologies with the cloud native world. Our focus is to place a kubernetes pod into a TEE. 
```

Comment:
In the first sentence do we want to mention 1) Transparent deployment of unmodified containers 2) Support for multiple hardware platforms 3) A strict trust model that separates the CSP from guest applications

Reply:
Thanks, updated this section to try annd include more detail around this